panther_driver/src/panther_can.py

```diff
@@ -104,6 +105,24 @@ def query_battery_data(self) -> Generator:
                 
                 for battery_data in motor_controller.battery_data:
                     yield battery_data
+                    
+    def query_driver_temperature_data(self) -> Generator:
+        with self._lock:
+            for motor_controller in self._motor_controllers:
+                try:
+                    # division by 10 is needed according to documentation
+                    motor_controller.temperature = \
+                        float(
+                            motor_controller.can_node.sdo['Qry_TEMP'][0].raw
+                        )
```

Comment:
Is it  or not? I don't see it divided by 10. 

Moreover, the temperature of the CPU controller is read. Isn't it better to read the temperature of individual channels? More information can be found [here](https://www.roboteq.com/docman-list/motor-controllers-documents-and-files/documentation/user-manual/272-roboteq-controllers-user-manual-v21/file) on page 264.